idb/postgres/internal/writer/writer_test.go

```diff
@@ -410,6 +410,70 @@ func TestWriterTxnParticipationTableBasic(t *testing.T) {
 	}
 }
 
+func TestWriterTxnParticipationTableAppCallAddresses(t *testing.T) {
```

Comment:
Could you merge this with `TestWriterTxnParticipationTableBasic`? These tests appear to be identical except for the transactions and expected results, so they could be data-driven with one of those test case structs.

Reply:
done